smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/handlers/HttpHeaderMiddleware.kt

```diff
@@ -53,20 +54,24 @@ class HttpHeaderMiddleware(
                 .sortedBy { it.memberName }
             val prefixHeaderBindings = requestBindings
                 .filter { it.location == HttpBinding.Location.PREFIX_HEADERS }
+            val opIndex = OperationIndex.of(ctx.model)
+            val inputShape = opIndex.getInput(op).get()
+            val hasHeaders = MiddlewareShapeUtils.hasHttpHeaders(ctx.model, op)
```

Comment:
This variable `hasHeaders` does not seem to be used?

Reply:
corrected